nodejs/eks/cluster.ts

```diff
@@ -464,6 +464,11 @@ export interface ClusterNodeGroupOptions {
      * Custom k8s node labels to be attached to each woker node
      */
     labels?: { [key: string]: string };
+
+    /**
+     * Custom k8s node taints to be attached to each worker node
+     */
+    taints?: { [key: string]: string };
```

Comment:
Looking at this again - I believe this is not sufficient for `taints` - the `register-with-taints` flag expects something of the form:

```
--register-with-taints=node-role.kubernetes.io/master=true:NoSchedule
```

This has three parts:
* `key`
* `value`
* `effect`

arranged as `key=value:effect` and where `effect` is (currently) one of `NoSchedule`, `NoExecute` or `PreferNoSchedule`.

We could keep this as-is - and just document that the `value` here must be of the form `val:effect`.  But probably better to provide an actual data structure for this.